### ISSUE TITLE
[feature request] option to make widget not scrollable

### ISSUE THREAD

**Issue by victorjagu** [andstatus/todoagenda#148]
Hi there!
Lovely app 😊
May I kindly request to add an option to make the widget not scrollable? 
Tiny screen makes easy to scroll by accident... so sometimes I'm not aware that I'm not in "today" and I think I have no appointments 😅

Edit: or perhaps an intend to scroll all the way up with apps like Tasker? I don't know if it already exists 

Thanks!
Victor

**Comment by yvolk** [comment]
Hi all and @victorjagu 
Any volunteer to create a non-scrollable layout for the widget?!

**Comment by yvolk** [comment]
@victorjagu BTW we do have a small icon in the Widget's header clicking on which scrolls the widget to Today 
![image](https://github.com/user-attachments/assets/a4bf65eb-cba0-4de3-be8a-5f746e4a308c)

**Comment by victorjagu** [comment]
Yeah, I know @yvolk, however that does not fully solve my issue 😅
Anyway, thanks for the tip!

**Comment by yvolk** [comment]
The simplest way to implement what you need is to limit maximum number of events in a widget... It won't scroll just because it won't have any events not fit into it...

**Comment by NicolaSmaniotto** [comment]
Hi, I too would like an option to make the widget not scrollable (I don't want moving things in my launcher).

The maximum number of events is almost a workaround, however the "right" number changes based on the density of the events. Single days with many appointments will fill the widget on their own, while a week with few events has many date headers between them (and fills more vertical space).

Just limiting the number of events would either overflow in the first case or waste space in the second. I'd like the widget to use the available space as much as it can.

Could you please reopen this issue to make it more visible if someone wants to contribute?

**Comment by victorjagu** [comment]
Opened [PR#203](https://github.com/andstatus/todoagenda/pull/203) to implement this functionality. I hope this helps and that it is done correctly. Thanks for give it a try.

**Comment by yvolk** [comment]
@victorjagu I published beta 4.14.1 with this change https://github.com/andstatus/todoagenda/releases/tag/4.14.1-beta
